apps/frontend/src/renderer/components/MarkdownContent.tsx

```
@@ -0,0 +1,184 @@
+                  </code>
+                </pre>
+              </div>
+            );
+
+          case 'file-link':
+            return (
+              <button
+                key={index}
+                onClick={() => handleFileClick(block.filePath!, block.lineStart, block.lineEnd)}
+                className="inline-flex items-center gap-2 rounded-md bg-primary/10 px-3 py-1.5 text-sm font-medium text-primary hover:bg-primary/20 transition-colors"
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)